.gitignore

```
@@ -1,6 +1,9 @@
 # iPython checkpoint folders
 .ipynb_checkpoints/
 
+# Pickled solutions
+*.p
```

```suggestion
*.pkl
```

.pkl is extension suggested by python 2 docs: 
https://docs.python.org/2/library/pickle.html#example